android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/service/BdsService.java

```diff
@@ -100,17 +100,21 @@ private RequestResponse handleException(HttpURLConnection urlConnection,
   }
 
   public void makeRequest(String endPoint, String httpMethod, List<String> paths,
-      Map<String, String> queries, Map<String, Object> body,
+      Map<String, String> queries, Map<String, Object> body, boolean shouldParallelize,
```

Comment:
if you're gonna leave shouldParallelize flag here, you should also change its name on interface